### PR TITLE
Adds postgres backend

### LINKED ISSUE #71: Add psql server to docker-compose project

### PR DESCRIPTION
**Pull request by jlhogan**
should close #71 

and rider: removes ProblemDetails model, which is a holdover from .Net

### CODE REVIEW

File: src/models/problemDetails.ts [hunk 1]
```diff
@@ -1,8 +0,0 @@
-export interface ProblemDetails {
```
Comment (JordanGuinn) [inline src/models/problemDetails.ts:1]: 👏🏽👏🏽👏🏽

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
 		"http-proxy": "^1.18.1",
 		"jwks-rsa": "^1.8.1",
 		"multer": "^1.4.2",
+		"node-postgres": "^0.6.2",
```
Comment (JordanGuinn) [inline package.json:21]: Is `node-postgres` here just for this initial POC, or is this to say we'll be using drivers + raw SQL over an ORM (captured in #72)?

File: docker-compose.yaml [hunk 3]
```diff
@@ -30,6 +30,14 @@ services:
     environment:
       HOST: '0.0.0.0'
       NODE_ENV: development
+      DB_PASSWORD: TestPassword1
+    depends_on:
+      - db
+
+  db:
+    image: postgres:13
```
Comment (JordanGuinn) [inline docker-compose.yaml:38]: Well that was easy 🎉

File: package.json (hunk 2 above)
Comment (jlhogan) [reply package.json:21]: just here for now to :heavy_check_mark: that we're indeed connected to a working postgres serverr

**Pull request merged**